doc/cephadm/services/osd.rst

```diff
@@ -144,6 +144,18 @@ There are a few ways to create new OSDs:
 
     ceph orch daemon add osd host1:data_devices=/dev/sda,/dev/sdb,db_devices=/dev/sdc,osds_per_device=2
 
+* Create an OSD from a specific lvm on a specific host:
+
+  .. prompt:: bash #
+
+    ceph orch daemon add osd *<host>*:*<lvm-path>*
+
+  For example:
+
+  .. prompt:: bash #
+
+    ceph orch daemon add osd host1:/dev/vg_osd/lvm_osd
```

Comment:
suggest an example that is more reflective of a host having more than one OSD.  Something like

"ceph orch daemon add osd host1:/dev/vg_osd/lvm_osd1701"